examples/plot_fpga_lenet.py

```diff
@@ -0,0 +1,89 @@
+"""
```

Comment:
what do you mean by the name "plot" in the filename

Reply:
`plot_` is the default prefix used by sphinx gallery. We can change it if we want, suggestions welcome.

Reply:
ok, what do you think about `demo_`?